src/styles/less/variables.less

```diff
@@ -13,6 +13,6 @@
 @button-blue: #007aff;
 
 // Media Queries
-
+@xs: ~'(max-width:320px)';
```

Comment:
I really wanted to add more global media breakpoints also - especially one in the 500 range. Perhaps if it is something that becomes an issue in a few places we can talk about putting one in.